slsa-e2e-rfp/README.md

```diff
@@ -0,0 +1,249 @@
+# SLSA E2E RFP
+
```

Comment:
Would be good to add some text about what the demo shows e2e? i.e.,

We have some attestations, here's what the supply chain looks like, here's what the policy verified, etc.

Reply:
Yeah, Snahil and I were discussing this today, and we think it's best to keep the [writeup](https://docs.google.com/document/d/1sbJfhVdjcPJIH3_m1EMZlxuw9FAqV12n4aCvNg2hcbw/edit?usp=sharing) in a separate Gdoc for now for easier commenting.